common/src/main/nl/jft/common/util/elo/EloCalculationStrategy.java

```diff
@@ -0,0 +1,19 @@
+package nl.jft.common.util.elo;
+
+/**
+ * An {@code EloCalculationStrategy} is a strategy to calculating a new Elo-rating based on the given parameters.
+ *
+ * @author Oscar de Leeuw
+ */
+public interface EloCalculationStrategy {
```

Comment:
![CRITICAL](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-critical.png) Annotate the "EloCalculationStrategy" interface with the @FunctionalInterface annotation (sonar.java.source not set. Assuming 8 or greater.) [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://92.110.24.119:9000/coding_rules#rule_key=squid%3AS1609)